examples/terraform/wordpress/metadata.display.yaml

```diff
@@ -117,6 +120,13 @@ spec:
         source_image:
           name: source_image
           title: Source Image
+          enumValueLabels:
```

Comment:
Can we use normal yaml syntax here? It also might be worth adding a second label-value pair for illustration.